Source/WebCore/platform/graphics/Color.cpp

```diff
@@ -49,6 +49,51 @@ Color::Color(Color&& other)
     *this = WTFMove(other);
 }
 
+Color::Color(ColorData&& colorData)
+{
+    OptionSet<FlagsIncludingPrivate> flags;
+    if (colorData.isSemantic)
+        flags.add(FlagsIncludingPrivate::Semantic);
+    if (colorData.usesFunctionSerialization)
+        flags.add(FlagsIncludingPrivate::UseColorFunctionSerialization);
+
+    if (std::holds_alternative<PackedColor::RGBA>(colorData.data)) {
+        flags.add({ FlagsIncludingPrivate::Valid });
+
+        auto& d = std::get<PackedColor::RGBA>(colorData.data);
+        setColor(asSRGBA(d), flags);
+    } else if (std::holds_alternative<OutOfLineColorData>(colorData.data)) {
+        flags.add({ FlagsIncludingPrivate::Valid, FlagsIncludingPrivate::OutOfLine });
+
+        auto& d = std::get<OutOfLineColorData>(colorData.data);
+        setOutOfLineComponents(OutOfLineComponents::create({ d.c1, d.c2, d.c3, d.alpha }), d.colorSpace, flags);
+    }
+}
+
+Color::ColorData Color::data() const
+{
+    if (isOutOfLine()) {
+        auto& outOfLineComponents = asOutOfLine();
+        auto [c1, c2, c3, alpha] = outOfLineComponents.unresolvedComponents();
+
+        OutOfLineColorData oolcd = {
+            .colorSpace = colorSpace(),
+            .c1 = c1,
+            .c2 = c2,
+            .alpha = alpha
+        };
+
+        return { .isSemantic = flags().contains(FlagsIncludingPrivate::Semantic),
+                 .usesFunctionSerialization = flags().contains(FlagsIncludingPrivate::UseColorFunctionSerialization), // NOLINT
```

Comment:
Don't use NOLINT here.  Just make the indentation more like this:
return {
    something,
    somethingElse
};